recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.5.3" %}
+{% set variant = "openblas" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 200
+    skip: true    # [win and not py35]
+    features:
+        - vc14                # [win and py35]
+        - blas_{{ variant }}  # [unix]
+
+# NOTE: This installs the minimum dependencies for Simbody, i.e., no
+# visualizer. blas is the only required dependency, as it will install openblas
+# which also includes lapack. blas is not needed for Windows because Simbody
+# includes blas/lapack for the Windows builds.
+# TODO: Once FreeGLUT, Xi, and Xmu are all avaliable via Anaconda.org (free or
```

Comment:
Agreed, I doubt anyone is build Xorg. Hence the recommendation to us `yum_requirements.txt`. All of this is Linux specific.

Mac doesn't really ship with an X windowing system. There use to be XQuartz and it still exists, but it no longer comes pre-installed. So we can't really make any assumptions about whether that will be available or not. Building XQuartz is no less complicated that building Xorg. We only support `freeglut` on Linux.